kernels/test/op_leaky_relu_test.cpp

```diff
@@ -40,10 +40,29 @@ class OpLeakyReluTest : public OperatorTest {
     EXPECT_TENSOR_EQ(out, ret);
     EXPECT_TENSOR_EQ(out, tf.ones({2, 2}));
   }
+
+  template <ScalarType DTYPE>
+  void expect_bad_scalar_value_dies(const Scalar& bad_value) {
+    TensorFactory<DTYPE> tf;
+    Tensor in = tf.ones({2, 2});
+    Tensor out = tf.zeros({2, 2});
+
+    ET_EXPECT_KERNEL_FAILURE(context_, op_leaky_relu_out(in, bad_value, out));
+  }
 };
 
 TEST_F(OpLeakyReluTest, SanityCheck) {
 #define TEST_ENTRY(ctype, dtype) test_leaky_relu_dtype<ScalarType::dtype>();
   ET_FORALL_FLOATHBF16_TYPES(TEST_ENTRY);
 #undef TEST_ENTRY
 }
+
+TEST_F(OpLeakyReluTest, FloatTensorTooSmallScalarDies) {
```

Comment:
wasn't there shared infrastructure for this?

Reply:
leaky_relu is different than the others, it only accepts floating point scalars. The shared macro tests integrat dtypes as well, so I cannot use it here.